pkg/agent/qrm-plugins/cpu/dynamicpolicy/policy_allocation_handlers.go

```diff
@@ -911,20 +922,37 @@ func (p *DynamicPolicy) generatePoolsAndIsolation(poolsQuantityMap map[string]in
 		proportionalPoolsQuantityMap := make(map[string]int)
 
 		for poolName, poolQuantity := range poolsQuantityMap {
-			proportionalSize := general.Max(getProportionalSize(poolQuantity, poolsTotalQuantity, availableSize), 1)
+			proportionalSize := general.Max(getProportionalSize(poolQuantity, poolsTotalQuantity, availableSize, true /*ceil*/), 1)
```

Comment:
the comment `ceil` means `true` will ceil the result, and the `false` is not?

Reply:
yes